src/components/QuizEdit.vue

```diff
@@ -52,6 +59,36 @@ const isSelecting = ref(false);
 const selectionStart = ref({ x: 0, y: 0 });
 const selectionEnd = ref({ x: 0, y: 0 });
 const hintPosition = ref({ x: 0, y: 0 });
+const currCursorPos = ref({ x: 0, y: 0 });
+const selectionStyle = computed(() => {
+  const getMinAlignment = (a: number, b: number) => {
```

Comment:
These 2 are essentially the same function - you could simplify to one by passing in a string/enum of "min" or "abs" or boolean?
```ts
const getDimension = (a: number, b: number, isMin: boolean) => {
  return `${(isMin ? Math.min(a, b) : Math.abs(a - b))}px`;
};
```

Reply:
fixed.